doc/related_projects.rst

```diff
@@ -200,6 +226,9 @@ Other packages useful for data analysis and machine learning.
 - `Deep Learning <http://deeplearning.net/software_links/>`_ A curated list of deep learning
   software libraries.
 
+- `sklearn-deap <https://github.com/rsteca/sklearn-deap>`_ Use evolutionary
+   algorithms instead of gridsearch in scikit-learn.
```

Comment:
gridsearch -> grid search.

This does not belong here. Perhaps a "model selection and evaluation" section under "other estimators and tasks"